drivers/wasapi/audio_driver_wasapi.cpp

```diff
@@ -559,7 +559,7 @@ Error AudioDriverWASAPI::init() {
 
 	Error err = init_output_device();
 	if (err != OK) {
-		ERR_PRINT("WASAPI: init_output_device error");
+		ERR_FAIL_V_MSG(ERR_CANT_OPEN, "WASAPI: init_output_device error");
```

Comment:
```suggestion
		ERR_FAIL_V_MSG(ERR_CANT_OPEN, "WASAPI: init_output_device error.");
```
While we're at it let's correct the format 🙂

Reply:
Or rather replace the whole block with:
```cpp
ERR_FAIL_COND_V_MSG(err != OK, ERR_CANT_OPEN, "WASAPI: init_output_device error.");
```
Or better yet:
```cpp
ERR_FAIL_COND_V_MSG(err != OK, err, "WASAPI: init_output_device error.");
```

Reply:
makes sense! Thank you!